src/Components/Main/Pages/ProfilePage/Posts/Posts.js

```diff
@@ -0,0 +1,28 @@
+import classes from './Posts.module.scss';
+import { Post } from '../Post/Post';
+import { Button } from '../../../../Common/Button/Button';
+
+export const Posts = ({posts, postText, setPost, addPost}) => {
+  const postsElements = posts
+    .map((post) => <Post postText={post.postText} key={post.id} />);
+
+  const onSetPost = (event) => {
+    setPost(event.target.value);
+  };
+  const onAddPost = () => {
+    addPost();
```

Comment:
As I see "addPost" expects text as a prop here
 src/Components/Main/Pages/ProfilePage/Posts/PostsContainer.js
But you are not passing any text, is it expected?

Reply:
That is right!
Adding a post occurs in profileReducer in "case ADD_POST".